modules/pmi/pyext/src/restraints/residue_proximity_restraint.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import print_function
+import IMP
+import IMP.core
+import IMP.algebra
+import IMP.atom
+import IMP.container
+import IMP.pmi.tools
+import IMP.pmi.restraints
+
+import math
+import numpy as np
+
+class ResidueProteinProximityRestraint(IMP.pmi.restraints.RestraintBase):
+
+    """ Restraint residue/residues to bind to unknown location in a target"""
+
+    def __init__(self,
+                 hier,
+                 selection,
+                 cutoff = 6.,
+                 sigma = 3.,
+                 xi = 0.01,
+                 resolution=1.0,
+                 weight = 1.0,
+                 label = None):
+        """
+        Constructor
+        @ param hier        Hierarchy of the system
```

Comment:
I don't think doxygen will read `@ param`, needs to be `@param` without the space.